metabase/metabase
# Add popular section to the new homepage

Epic: https://github.com/metabase/metabase/issues/21322

The backend is not ready for this yet, but this PR is safe to be merged without it.

How to test:
- Change https://github.com/metabase/metabase/blob/b98fc2e09647328fe447963d65d32faa9e901d96/frontend/src/metabase/home/homepage/components/HomeContent/HomeContent.tsx#L22 to `return <HomePopularSection />;`
- Change https://github.com/metabase/metabase/blob/b98fc2e09647328fe447963d65d32faa9e901d96/frontend/src/metabase/home/homepage/containers/HomePopularSection/HomePopularSection.tsx#L4` to `export default RecentViews.loadList({ listName: "popularViews" })(HomePopularSection);`
- You should see a different view for popular items (= in the company) instead of recent items (= your own)

<img width="796" alt="Screenshot 2022-04-04 at 17 30 24" src="https://user-images.githubusercontent.com/8542534/161566334-60296606-79c7-4bbb-8f14-dd74052249f3.png">
